src/views/token-swap/components/dropdown/dropdown.view.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import useDropdrownpStyles from './dropdown.style'
+import { ReactComponent as Search } from '../../../../images/icons/search.svg'
+import { getTokenIcon } from '../../../../utils/tokens'
+import { getFixedTokenAmount } from '../../../../utils/currencies'
+
+function Dropdown ({ accounts, position, setToken, close }) {
+  const classes = useDropdrownpStyles()
+  const onClickToken = account => {
+    close()
+    setToken({ [position]: account })
+  }
+  const renderList = accounts.map(account => {
+    const Icon = getTokenIcon(account.token.symbol)
+    return (
+      <div
+        key={account.token.symbol}
+        className={classes.tokenBox}
+        onClick={() => onClickToken(account)}
+      >
+        <div className={classes.tokenIcon}>
+          <Icon />
+        </div>
+        <div className={classes.tokenText}>
+          <p>{account.token.name}</p>
+          <p className={classes.symbol}> {account.token.symbol}</p>
+        </div>
+        <span className={classes.balanceText}>
+          {getFixedTokenAmount(account.balance, account.token.decimals)}{' '}
```

Comment:
To avoid the `{' '}` we normally use template strings {`${getFixedTokenAmount(account.balance, account.token.decimals)} ${{account.token.symbol}}`}